roles/inspec/defaults/main.yml

```diff
@@ -14,6 +14,13 @@ inspec:
       name: inspec-stig-rhel7
       version: master
       url: https://github.com/inspec-stigs/inspec-stig-rhel7/archive/master.tar.gz
+    ursula-inspec:
+      name: ursula-inspec
+      version: master
+      url: https://github.com/blueboxgroup/ursula-inspec/archive/master.tar.gz
+      sub_profiles:
+        - name: inspec-mysql
+          path: /ursula-inspec/mysql
```

Comment:
I'm not sure how to put sub_profiles into profile without doing something overly complicated.
It gets used here:
https://github.com/blueboxgroup/ursula/blob/master/roles/inspec/templates/etc/inspec/host-controls/inspec.yml#L15

One thing I was thinking about was to iterate through inspec.controls.<item>, check if the <item> is true and then do a look up on the profile_location and fill out the dependencies in the inspec.yml template with those profiles. The only problem is that there are multiple items under inspec.controls that have the same profile (like horizon, keystone). So that would cause inspec to break (since you can't have duplicate dependency profile entries).